### ISSUE TITLE
[Bug]:  return nothing when calling API  by client.predict(

### ISSUE THREAD

**Issue by yixiyanyu** [lllyasviel/Fooocus#2778]
### Checklist

- [X] The issue has not been resolved by following the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md)
- [ ] The issue exists on a clean installation of Fooocus
- [X] The issue exists in the current version of Fooocus
- [ ] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

when calling result = client.predict(, it returns an empty tulpe.
I checked the  async_worker.py, there is no echo infomation which means the funciton was not called in anyway.
@torch.no_grad()
    @torch.inference_mode()
    def handler(async_task):
        execution_start_time = time.perf_counter()
        async_task.processing = True

        args = async_task.args
        args.reverse()

        print(f"***************handler {__file__}:{args}")

### Steps to reproduce the problem

client = Client(src=url, serialize=True, max_workers=max_workers)
def request_fooocus_predict(self, prompt, seed):
        return self.client.predict(
            # None,
            False,
            prompt,
            "",
            ["Fooocus Cinematic"],
            "Extreme Speed",
            "1280×768 <span style=\"color: grey;\"> ∣ 5:3</span>",
            1,
            "png",
            seed,
            True,
            2,
            4,
            "juggernautXL_v8Rundiffusion.safetensors",
            "None",
            0.5,
            True,
            "None",
            1,
            True,
            "None",
            1,
            True,
            "None",
            1,
            True,
            "None",
            1,
            True,
            "None",
            1,
            False,
            "uov",
            "Disabled",
            None,
            [],
            None,
            "",
            None,
            True,
            True,
            True,
            1.5,
            0.8,
            0.3,
            7,
            "dpmpp_2m_sde_gpu",
            "karras",
            -1,
            -1,
            -1,
            -1,
            -1,
            -1,
            False,
            False,
            False,
            False,
            64,
            128,
            "joint",
            0.25,
            True,
            1.01,
            1.02,
            0.99,
            0.95,
            True,
            True,
            "v2.6",
            1,
            0.618,
            False,
            False,
            0,
            False,
            "fooocus",
            None,
            0.5,
            0.6,
            "ImagePrompt",
            None,
            0.5,
            0.6,
            "ImagePrompt",
            None,
            0.5,
            0.6,
            "ImagePrompt",
            None,
            0.5,
            0.6,
            "ImagePrompt",
            fn_index=40,
        )

 logger.info(f"Result: {job}")
=========
the response is：
Result：()

### What should have happened?

The v2.1.862 works fine， but the latest version dows not work.

### Where are you running Fooocus?

Locally

### What operating system are you using?

CentOS 8

### Console logs

```Shell
No logs in server, and empty tulpe responsed.
```
